index.js

```diff
@@ -1,88 +1,116 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+let counter = 1 
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+const = Keith
+console.log(name)
+=> "Raffy"
+name = "Not Raffy" 
+=> TypeError
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red")
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+function drink_water(current_thirst_level) {
+  console.log("Man I sure am thirsty")
+  current_thirst_level -= 1 
+  console.log("Ahh that hits the spot")
+  return current_thirst_level
+}
```

Comment:
Well done. For Javascript, it's standard to use camel case for function and variable names. So instead of `current_thirst_level`, name it `currentThirstLevel`